designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -248,21 +239,19 @@ The following sources were used:
 
 Project `ClassLibrary1` uses the following vulnerable packages
    [netcoreapp2.0]:
-   Top-level Package              Requested  Resolved    Vulnerability
-   > My.Vulnerable.Package        2.0.0      2.0.0       https://URL-to-vuln-details
-   > My.Fixed.Package             1.1.0      1.1.0       https://URL-to-vuln-details
+   Top-level Package              Requested  Resolved    Severity   Vulnerability
+   > My.Vulnerable.Package        2.0.0      2.0.0       Low        https://URL-to-vuln-details
+   > My.Fixed.Package             1.1.0      1.1.0       Critical   https://URL-to-vuln-details
 
  > To see transitive packages, add the `--include-transitive` option.
- > To see prerelease packages, add the `--include-prerelease` option.
 ```
 
-*Combining the `--vulnerable` option with the `--offline` options is not supported. When doing so, the following error message will be presented:*
+Notes:
+* *Important! Installed packages that are deprecated or outdated but not vulnerable will not be part of this command's output.*
 
-```shell
-> dotnet list package --vulnerable --offline
+* None of the markers `(D)`, `(O)` or `(V)` are used in this report.
 
-Invalid command. Combining the '--vulnerable' and '--offline' options is not supported.
-```
+* The Severity values will be colored default for Low, yellow for Moderate, and red for High and Critical (note that we can't represent this in GitHub markdown above).
 
 *Combining the `--vulnerable` option with the `--deprecated` option is (currently) not supported. When doing so, the following error message will be presented:*
```

Comment:
> the following error message will be presented:

where is the error message?

Reply:
Following this line is this shell text:

```shell
> dotnet list package --vulnerable --outdated

Invalid command. Combining '--vulnerable' and '--outdated' options is not supported.
```

That's the error referred to by this comment.